scripts/docs/package.json

```
@@ -3,15 +3,28 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "generate-cli-docs": "tsx ./src/generate-cli-docs.ts"
+    "generate-cli-docs": "tsx ./src/generate-cli-docs.ts",
+    "generate-config-docs": "tsx ./src/generate-config-docs.ts",
+    "test": "vitest",
+    "test:run": "vitest run",
```

Can we please keep the `test` script non-interactive:

```suggestion
    "test": "vitest run",
    "test:watch": "vitest",
```

It is often run in pre-commit hooks where it is desirable to fail the run on test failure, not watch/wait for changes (eg. in git clients with GUI).